src/routes/SignIn/index.jsx

```
@@ -9,9 +9,12 @@ import {
 import { Input, Button } from "@/shared/components";
 import logo from "@/assets/logo.png";
 import { useNavigate } from "react-router";
+import { useContext } from "react";
+import { loginContext } from "@/shared/context/loginContext";
 
 export default function Signin() {
   const navigate = useNavigate();
+  const { isLogin, setIsLogin, setUser } = useContext(loginContext);
```

ESLint

[error] 'isLogin' is assigned a value but never used.

(no-unused-vars)